src/main/java/com/mturko/bestshop/model/Product.java

```diff
@@ -1,21 +1,25 @@
 package com.mturko.bestshop.model;
 
-import org.springframework.data.annotation.Id;
-
+import javax.persistence.*;
 import java.time.LocalDateTime;
 
-public class Product {
+@Entity
+@Table(name = "product")
+ public class Product {
```

Comment:
Please overwrite `hashcode` and `equals` methods as described on the following page: https://docs.jboss.org/hibernate/stable/core.old/reference/en/html/persistent-classes-equalshashcode.html

I would suggest to move this logic to a common base abstract class, named like EntityObject or something similar

Reply:
Done